src/styles/main.scss

```diff
@@ -1,3 +1,203 @@
+
+  &__list {
+    display: flex;
+    align-items: center;
+  }
+
+  &__item {
+    list-style: none;
+    margin-right: 20px;
+  }
+
+  &__item:last-child {
+    margin: 0;
+  }
+
+  &__link {
+    text-decoration-line: none;
+    color: #060b35;
+    cursor: pointer;
+    padding: 23px 0;
+    position: relative;
+  }
+
+  &__link:hover {
+    color: #00acdc;
+  }
+
+  .nav__link.is-active {
+    color: #00acdc;
+  }
+
+  .nav__link.is-active::after {
+    content: " ";
+    display: block;
+    height: 4px;
+    width: 37px;
+    border-radius: 8px;
+    background-color: #00acdc;
+    position: absolute;
+    bottom: 0;
+  }
+}
+
+.catalog {
+  display: flex;
+  justify-content: center;
+  padding: 50px 0;
+  &__content {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+    max-width: 950px;
+  }
+}
+
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  font-style: normal;
+  color: #060b35;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-right: 48px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+  &:nth-child(-n+4) {
+    margin-bottom: 46px;
+  }
+
+  &__image {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 40px;
+  }
+
+  &__image-img {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    color: #616070;
+    margin-bottom: 16px;
+  }
+
+  &__rewievs,
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__rewievs {
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    color: #616070;
+  }
+
+  &__price-title {
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__link {
+    width: 166px;
+    height: 40px;
+    background: #00acdc;
+    border-radius: 5px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    color: #fff;
+  }
+
+  &__link:hover {
+    background: #fff;
+    border: 1px solid #00acdc;
+    color: #00acdc;
+  }
+}
+
+.stars {
+  display: flex;
+  &__star {
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-image: url("../images/star.svg");
+  }
+}
+
+.stars--5 > .stars__star,
+.stars--1 > .stars__star:nth-child(1),
+.stars--2 > .stars__star:nth-child(-n + 2),
+.stars--3 > .stars__star:nth-child(-n +3),
+.stars--4 > .stars__star:nth-child(-n +4) {
+  background-image: url("../images/star-active.svg");
+}
```

Comment:
Due to the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)
![image](https://user-images.githubusercontent.com/80700247/225332633-dbba6d71-16a3-4e1a-8725-9694bfad9c9d.png)